### DISCUSSION TITLE
Does anyone know what happened here?

### DISCUSSION THREAD

**Discussion by DylanZDD** [magicpdf/Magic-PDF#730] [Q&A]
<img width="389" alt="Screenshot 2024-10-12 at 20 05 07" src="https://github.com/user-attachments/assets/bfb1b05f-46f0-495e-a1d1-a306a633fcac">

**Comment by myhloli** [comment]
It might be due to dependency incompatibility that version 0.6.1 of magic-pdf was installed. What you need to do is create a Python 3.10 environment using conda and install the application with the commands provided in the README.
